tests/snapshots/TestCLICommands_Valid_log_file_in_env_should_be_priortized_over_config.stdout.golden

```diff
@@ -1,6 +1,23 @@
+DEBU start process loading config...
+DEBU start process loading embedded config...
+DEBU Failed to find config file on system path '/usr/local/etc/atmos/atmos'
+DEBU Failed to find config file on system path '/etc/atmos'
+DEBU No system config file found
+DEBU Check atmos config current working directory: /absolute/path/to/repo/
+DEBU Failed to find work dir atmos config file in path '/absolute/path/to/repo/atmos.d/**/*'.error no files matching name `atmos` with extensions [.yaml,.yml]  found in the provided directory: /absolute/path/to/repo/atmos.d/**/*
+DEBU Failed to find config file in path '/absolute/path/to/repo/atmos.d/**/*'
+DEBU Failed to find work dir atmos config file in path '/absolute/path/to/repo/.atmos.d/**/*'.error no files matching name `atmos` with extensions [.yaml,.yml]  found in the provided directory: /absolute/path/to/repo/.atmos.d/**/*
+DEBU Failed to find config file in path '/absolute/path/to/repo/.atmos.d/**/*'
+DEBU Failed to get work dir atmos config paths: no files matching name `atmos` with extensions [.yaml,.yml]  found in the provided directory: /absolute/path/to/repo/.atmos.d/**/*
+DEBU Found work dir atmos config file: /absolute/path/to/repo/atmos.yaml
+DEBU atmos merged config file /absolute/path/to/repo/atmos.yaml
+DEBU Workdir atmos loaded
+DEBU No configuration found in user preferences
+DEBU base path from infra atmos: /absolute/path/to/repo/atmos.yaml
+DEBU base path from infra: /absolute/path/to/repo/
 DEBU Found ENV var ATMOS_LOGS_FILE=/dev/stdout
 DEBU Using command line argument '--logs-level=Debug'
 DEBU processStoreConfig atmosConfig.StoresConfig=map[]
```

Comment:
All the DEBU logs going to `stdout` must mean this snapshot was generated without being updated to `main`